### DISCUSSION TITLE
pikpak不能挂载了

### DISCUSSION THREAD

**Discussion by ririye** [AlistGo/alist#7352] [Q&A] [answered]
alist版本是3.38.0，昨晚还好好的，今天pikpak就用不了了，显示Post "https://user.mypikpak.com/v1/auth/token?client_id=YNxT9w7GMdWvEOKa": dial tcp: lookup user.mypikpak.com: no such host，删除旧的挂载重新弄也是同样问题

**Comment by kafuneri** [comment, marked as answer]
pikpak的dns解析错误了，谷歌的8.8.8.8和cf的1.1.1.1都解析不了pikpak，OpenDNS的208.67.222.222 208.67.220.220还能用，现在cf代理都用不了，只能本机代理，离谱

**Reply by ririye** [reply]
感谢，换了这俩dns好啦
